api/lib/application/organization-invitations/organization-invitation-controller.js

```diff
@@ -1,6 +1,7 @@
 const _ = require('lodash');
+
+const httpErrors = require('../http-errors');
```

Comment:
on peut destructurer pour alléger l'usage dans le fichier.
```suggestion
const { MissingQueryParamError } = require('../http-errors');
```

Reply:
Merci, on peut destructurer aussi à d'autres endroits, done.